packages/create-gasket-app/lib/utils/warn-if-outdated.js

```diff
@@ -1,10 +1,12 @@
-const { runShellCommand } = require('@gasket/utils');
-const semver = require('semver');
-const chalk = require('chalk');
-const { readFile, writeFile } = require('fs/promises');
-const path = require('path');
+import { default as gasketUtils } from '@gasket/utils';
```

Comment:
Couldn't this be
```
import gasketUtils from '@gasket/utils';
```

or rather
```
import { runShellCommand } from '@gasket/utils';
```

Reply:
Minor observation as this file is going to be deduped with https://github.com/godaddy/gasket/pull/737